src/main/java/core/basesyntax/service/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private FruitStorageDao storageDao;
+
+    public ReturnOperation(FruitStorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction) {
+        storageDao.getAllFruits()
+                .merge(transaction.getFruit(), transaction.getQuantity(), Integer::sum);
```

Comment:
Nice use of the merge method to add the quantity of fruit returned. It's a clean and concise way to update the storage.